google/resource-snippets/pubsub-v1/pubsub.jinja

```diff
@@ -14,48 +14,30 @@
 
 resources:
   - type: pubsub.v1.topic
-    name: {{ properties["topic_name"] }}
+    name: {{ properties["topicName"] }}
     properties:
       topic: my-pubsub-topic
-  - type: pubsub.v1.topic
-    name: {{ properties["backup_topic_name"] }}
-    properties:
-      topic: my-backup-topic
   - type: pubsub.v1.subscription
-    name: {{ properties["subscription_name"] }}
+    name: {{ properties["subscriptionName"] }}
     properties:
       subscription: my-pubsub-subscription
```

Comment:
The comment above about `{{ properties["topicName"] }}` applies to this as well

Reply:
See my comment above in response to {{ properties["topicName"] }}